helper-apps/cortex-file-handler/blobHandler.js

```diff
@@ -142,12 +142,14 @@ async function uploadBlob(
       });
 
       busboy.on("file", async (fieldname, file, info) => {
-        //set useGoogle if file is image or video
+        //do not use google if file is image or video
```

Comment:
I don't think I understand this.  Images and Videos are the types we want to store in both GCS and Azure.  I think all of our video cases are Google models and some of our images cases are in Azure and some in Google.  What am I missing?